tests/installation/d_installer/d_installer_cli_setup.pm

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi;
+use utils;
+
+sub set_root_disk {
```

Comment:
(nitpick) we are trying to avoid to create functions inside test, so we can evaluate better the complexity of them, for this case is readable as it is.
For other cases this degenerates in libraries that we don't need to have or maintain. It is about being less DRY at the test layer.

Reply:
Ah, I like writing scripts this way, it's more self-explanatory. But I can also just put comments.